lib/timeline.js

```diff
@@ -31,7 +31,8 @@ Timeline.Pin = Pin;
  */
 Timeline.prototype.sendUserPin = function (userToken, pin, callback) {
   if (typeof userToken !== 'string') {
-    return callback(new Error('Expected userToken to be a string.'));
+    process.nextTick(callback(new Error('Expected userToken to be a string.')));
```

Comment:
Yeah. I don't know what I was thinking. I appear to have lost the ability to JavaScript properly. Too much time spent doing Ruby I think.